.github/workflows/config/spelling_allowlist.txt

```diff
@@ -279,6 +299,7 @@ superpositions
 symplectic
 tablegen
 templated
+thres
```

Comment:
I don't think this is a word and therefore should probably not be added to the allowlist.